### DISCUSSION TITLE
pac admin copy should support environment security groups

### DISCUSSION THREAD

**Discussion by jordanabakerafs** [microsoft/powerplatform-build-tools#1039] [Ideas]
**Current Behavior**:
Currently `pac admin copy` does not appear to support setting an environment security group when running the operation.

This is supported in `pac admin create` with the `--security-group-id -sgid` parameters.

Security group addition is also supported in Powerapps Admin Powershell for copy operations, therefore we assume this functionality is available on the API.

Expected Behavior:
`pac admin copy` should expose `--security-group-id -sgid` parameters similar to `pac admin create` in order assign a security group at copy time.

**Comment by bryansonofpatt** [comment]
yeah, I was struggling with this functionality too!!! Great ask @jordanabakerafs!
